### PR TITLE
Add option to filter users by no home facility

### LINKED ISSUE #8038: Avail the feature to filter out the users with No Home Facilities in User Module
Currently there's no provision to filter-out Users with no Home Facilities, availing this feature of filter out users with No Home facility should help in easy retrieve of Remote Doctors.
attaching supporting Screen recording for ref.

https://github.com/coronasafe/care_fe/assets/166367729/f8e7c09a-cc77-4ce2-970a-01350bdd104d

### PR DESCRIPTION
**Pull request by shivankacker**
Backend PR : https://github.com/coronasafe/care/pull/2332

---
## Proposed Changes

- Fixes https://github.com/coronasafe/care_fe/issues/8038
- Adds an option to filter users by no home facility

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
image

@skks1212 the filter is working but when you applied the none filter, and reopen the advance filters slide over, you won't see, the 'none' option selected in-home facility field

steps

1. open advance filters select none in home facility
2. apply the filters
3. again open the advance fields
4. the home facility field will be empty

**Comment by bodhish** [comment]
Can't we make it friendly? Make the test better than just "None". ?

**Comment by shivankacker** [comment]
@bodhish what about "No home facility"?

**Comment by bodhish** [comment]
@shivankacker how about "No home facility assigned"

File: src/Components/Common/FacilitySelect.tsx [hunk 1]
```diff
@@ -66,6 +68,9 @@ export const FacilitySelect = (props: FacilitySelectProps) => {
           name: text,
         });
 
+      if (allowNone)
+        return [{ name: "None", id: "NONE" }, ...(data?.results || [])];
```
Comment (rithviknishad) [inline src/Components/Common/FacilitySelect.tsx]: i18n

**Pull request merged**